scripts/create-component/create-component.ts

```diff
@@ -24,6 +24,13 @@ const convergedComponentPackages = Object.entries(getAllPackageInfo())
   )
   .map(([packageName]) => packageName);
 
+const packagePathsByPackageName = Object.entries(getAllPackageInfo())
+  .map(([packageName, packageInfo]) => [packageName, packageInfo.packagePath])
+  .reduce((obj, [packageName, packagePath]) => {
+    obj[packageName] = packagePath;
```

Comment:
nit: Why not remove the map?
```suggestion
  .reduce((obj, [packageName, packageInfo]) => {
    obj[packageName] = packageInfo.packagePath;
```

Reply:
not sure if this complexity is the right way to go. we have one source of truth of terms of path in our monorepo - workspace.json.

I'm leveraging that fact in following PR that fixes create-component https://github.com/microsoft/fluentui/pull/22877/files#diff-6f1893d8ae90dfa043a389e0feb82f9c8b248bf20b3e3f66dfcf3c8d6c122ea0R72

Reply:
I'd suggest to close this PR ?

Reply:
I was still working under the assumption that NX is not being used in these scripts... but I don't mind using workspace.json either

Reply:
we run our migration generator after every run. but yes it isn't used like to do the whole generation story. we plan to do that in the future